### PR TITLE
tbi(project-access): adds missing constants to DirName

### PR DESCRIPTION
**Pull request by mikicvi-SAP**
#1799 

- Adds missing constants entries that are required for app-gen, internal issue 28284.
- Cleanup existing constants, adp pulls constants from project-access

### CODE REVIEW

**Review by devinea — APPROVED** [review]
Code changes are clear and reduce duplication. 
Changeset ✅

File: packages/adp-tooling/src/preview/routes-handler.ts [hunk 1]
```diff
@@ -119,7 +120,7 @@ export default class RoutesHandler {
                 return;
             }
 
-            const fullPath = path.join(sourcePath, FolderTypes.CHANGES, FolderTypes.FRAGMENTS);
+            const fullPath = path.join(sourcePath, DirName.Changes, DirName.Fragment);
```
Comment (cianmSAP) [inline packages/adp-tooling/src/preview/routes-handler.ts:123]: `FolderTypes.FRAGMENTS` **plural** is changed to ` DirName.Fragment` **singular**
Is this expected?

File: packages/project-access/src/constants.ts [hunk 2]
```diff
@@ -15,6 +15,27 @@ export const FileName = {
     UI5DeployYaml: 'ui5-deploy.yaml'
 } as const;
 
-export const DirName = {
-    Changes: 'changes'
-} as const;
+export const enum DirName {
```
Comment (devinea) [inline packages/project-access/src/constants.ts:18]: Why was this changed to `const enum`?

File: packages/project-access/src/constants.ts [hunk 3]
```diff
@@ -15,6 +15,27 @@ export const FileName = {
...
+    Changes = 'changes',
+    Sapux = 'src',
```
Comment (devinea) [inline packages/project-access/src/constants.ts:20]:
```suggestion
```
No longer used. Should not be included here

File: packages/project-access/src/constants.ts (hunk 2 above)
Comment (devinea) [reply packages/project-access/src/constants.ts:18]: @Klaus-Keller This was a deliberate change in open source right?

File: packages/project-access/src/constants.ts (hunk 3 above)
Comment (mikicvi-SAP) [reply packages/project-access/src/constants.ts]: Adressed in [44d26ce](https://github.com/SAP/open-ux-tools/pull/1801/commits/44d26ce5a994fd469437d09d124d7141afe15ef6). Thanks

**Review by devinea — APPROVED** [review]
Thanks for the updates.

**Review by longieirl — COMMENTED** [review]
Why is there a change to the pnpm-lock.yaml file?

**Comment by longieirl** [comment]
> Why is there a change to the pnpm-lock.yaml file?

Sorry! You have pulled in "@sap-ux/project-access": "workspace:*", into the relevant project.

**Review by cianmSAP — APPROVED** [review]
Comments have been addressed
Changes look good
